src/core/tsi/ssl_transport_security.h

```diff
@@ -40,6 +40,15 @@ extern "C" {
 
 #define TSI_X509_PEM_CERT_PROPERTY "x509_pem_cert"
 
+// Please note that internally, we just faithfully pass whatever value we got by
```

Comment:
OK, I removed it.
I just thought adding a comment here would be a "reminder" for all other languages, otherwise if the field is exposed by other languages, we will need to remember to add similar comments.